### PR TITLE
Issue4

### PR DESCRIPTION
**Pull request by gtw2**
Updates PRIDE input file to match new PyRe structure. 

Fixes isotopic composition of input waste, and re-aligns waste streams to reflect those specific to PRIDE's design. 

PyRe runs and is recognized in the SQLite, however requires a record function to get valuable data out of each run. This is described in #7 and #8 .

This will close issue #4 .

### CODE REVIEW

File: input/pride.xml [hunk 1]
```diff
@@ -20,179 +31,202 @@
         <feed_commods> <val>leu</val> </feed_commods>
         <feed_commod_prefs> <val>1.0</val> </feed_commod_prefs>
         <feed_recipe>lwr_waste</feed_recipe>
+        <volox_temp>900</volox_temp>
```
Comment (katyhuff) [inline input/pride.xml]: May be worth indicating in comments where these numbers come from (perhaps these are estimates at the moment? They look to be very round numbers.

File: src/pyre.cc [hunk 2]
```diff
@@ -168,19 +172,16 @@ cyclus::Material::Ptr Pyre::Separate(Stream stream,
   Material::Ptr material;
   switch (stream_count) {
     case 1:
-    case 2:
       material = v->VoloxSepMaterial(stream.second, mat);
       break;
-    case 3:
-    case 4:
+    case 2:
```
Comment (katyhuff) [inline src/pyre.cc]: I still don't understand the meaning of these stream integers.

**Comment by gwenchee** [comment]
Does this input file run successfully in Cyclus?

File: input/pride.xml (hunk 1 above)
Comment (gtw2) [reply input/pride.xml]: Yeah they're estimates at the moment, good point though I'll comment the values given

File: src/pyre.cc (hunk 2 above)
Comment (gtw2) [reply src/pyre.cc]: Just to mark that the first stream goes to volox, second goes to reduction, etc.

**Comment by gtw2** [comment]
@gwenchee yes, it runs

**Comment by katyhuff** [comment]
We should set up CI for this repo.

**Pull request merged**